cordova-lib/src/plugman/registry/registry.js

```diff
@@ -236,15 +231,22 @@ function initSettings() {
     module.exports.settings =
     rc('plugman', {
         cache: plugmanCacheDir,
-        registry: 'http://registry.cordova.io',
+        registry: registryURL,
         logstream: fs.createWriteStream(path.resolve(plugmanConfigDir, 'plugman.log')),
         userconfig: path.resolve(plugmanConfigDir, 'config'),
         'cache-min': oneDay
     });
+
+    // if npm is true, use npm registry. 
+    // ~/.plugman/config overides the above config if it exists. 
```

Comment:
I think it is by design. I don't know the reason though. 

As far as I can tell, `~/.plugman/config` is used by `plugman publish`. We probably don't actually want to change the registry value in the config itself. 

What do you mean by wouldn't you want to be able to still set this?